### PR TITLE
Provide access to contracts

### PR DESCRIPTION
**Pull request by ronaldour**
This will provide access to contracts by client class, thus this will provide access to severity levels #55 

@gabsanchez @ronaldour @childmario95 @tohka53 @mariopg @oscargarciacolon @gzepeda

### CODE REVIEW

**Comment by OsvaldoRosado** [comment]
Exposing this on `Client` seems like the wrong choice to me. Why not the root applicationinsights.ts?

**Comment by ronaldour** [comment]
we thought about the two options but you are right it would be better on applicationinsights.ts

**Comment by ronaldour** [comment]
Updated to access contracts by applicationinsights.ts  class

File: applicationinsights.ts [hunk 1]
```diff
@@ -22,6 +23,7 @@ class ApplicationInsights {
     * The default client.
     */
     public static client: Client;
+    public static contracts = Contracts;
```
Comment (OsvaldoRosado) [inline applicationinsights.ts]: Do you think this should be capitalized? I know all of the other fields are lowercase, but this isn't a field so much as exposing a namespace. @KamilSzostak does JS SDK have any precedence on this to follow?

Comment (KamilSzostak) [inline applicationinsights.ts]: Not really. The only problem I had was with source maps, which is case-sensitive for file paths, but this is not the case here.

**Comment by gzepeda** [comment]
Kudos to the team!

**Pull request merged**